modules/styling/stories/HowToStyle.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import {CustomButton} from './examples/CustomButton';
+import {CustomIcon} from './examples/CustomIcon';
+import {StyledButton} from './examples/StyledButton';
+
+<Meta title="Styling/ How to Customize Styles" />
+
+# How To Customize Styles
+
+There are multiple ways to customize styles for our components within Canvas Kit. What approach you
+choose depends on your use case. Ranging from some simple overrides to fully custom solutions, here
+are the following options.
+
+- [Styles](#styles)
+- [Stencils](#stencils)
+
+## Styles
+
+### Use `createStyles` With `cs` If
+
+Use `createStyles` in tandem with `cs` prop when you're overriding static styles and making small
+modifications to an existing Canvas Kit component like padding, color and flex properties. Take our
+`Text` component as an example.
+
+```tsx
+import {createStyles} from '@Workday/canvas-kit-styling';
+import {system} from '@Workday/canvas-tokens-web';
+import {Text} from '@Workday/canvas-kit-react/text';
+
+const uppercaseTextStyles = createStyles({
+  textTransform: 'uppercase',
+  margin: system.space.x4
+})
+//...
+<Text cs={uppercaseTextStyles}>My uppercased text</Text>;
+```
+
+> **Note:** `createStyles` handles wrapping our token variables in `var(--${tokenName})`
+
+## Stencils
+
+### Use Stencils If
+
+Stencils help you organize the styling of reusable components into base styles, modifiers, and
+variables. The organization makes it more natural to produce static and clean CSS with optional
+extraction into CSS files.
+
+```tsx
+export const buttonStencil = createStencil({
+  vars: {
+    backgroundColor: '',
+    textColor: '',
+  },
+  base: ({backgroundColor, textColor}) => ({
+    backgroundColor: cssVar(backgroundColor, base.berrySmoothie400),
```

Comment:
Do these need `cssVar`?

Reply:
yes, they need it as fallback for variable

Reply:
this example doesn't extend `buttonStencil` so there's no default. So we either add the extends and remove this, or keep as is.

Reply:
Thanks for the clarification!